### DISCUSSION TITLE
Release v2.11.0-preview.1

### DISCUSSION THREAD

**Discussion by bruth** [nats-io/nats-server#5138] [General]
## 📣 Notice
This is the first **preview release** of 2.11.0! The motivation is to release new features and changes during the development lifecycle. This release is **not feature complete** for what the final 2.11.0 release will contain. This should only be used in **development environments** and **not** production. Additional preview releases will become available in subsequent weeks. Please check it out and give us feedback!

### Go Version
- 1.22.0 (#5133)

### Added

Core
- Distributed pub/sub message tracing including OpenTelemetry support (#5014, #5057, #5078, #5121)
  - Refer to the [ADR](https://github.com/nats-io/nats-architecture-and-design/blob/main/adr/ADR-41.md) for more details

JetStream
- Support for batched direct-gets requests from streams (#4981, #5036)
  - Refer to the [ADR](https://github.com/nats-io/nats-architecture-and-design/blob/main/adr/ADR-31.md#batched-requests) for more details
- Add support for pausing/resuming consumers (#5066)
- Support for batched direct multi-get requests from streams (#5107)
  - Refer to the (in-progress) [ADR](https://github.com/nats-io/nats-architecture-and-design/blob/4ceef38f300f3aa2cf8c8597788995c9dd015f20/adr/ADR-31.md#multiple-subject-requests) for more details

### Changed

MQTT
- Change retained messages to use headers for metadata rather and JSON-encoding the message (#5050)
  - Note this is currently a backwards incompatible change with 2.10.x servers.

### Improved

General
- Track recovery from slow consumer events in routes (#5101)

OCSP
- Make OCSP request with POST method (#5109)

JetStream
- Prevent potential degradation of publish latency over time (#5116) Thanks to @stefanLeo for contributing the issue and the improvement!

### Fixed

JetStream
- Add explicit structs for the account claims update response payload (#4942)
- Acking a redelivered msg with more pending outstanding that should move the ack floor (#5008)

MQTT
- Header length calculation for stored retained messages (#5125)

### Complete Changes
 
https://github.com/nats-io/nats-server/compare/v2.10.11..v2.11.0-preview.1

<hr /><em>This discussion was created from the release <a href='https://github.com/nats-io/nats-server/releases/tag/v2.11.0-preview.1'>Release v2.11.0-preview.1</a>.</em>

**Comment by jhalag** [comment]
This appears to have #5131 included, however I don't see it in the notes?
